Vagrantfile

```diff
@@ -109,71 +77,32 @@ $build_id = "#{$job_name}-#{$build_number}"
 # we can run VMs locally without having any the `build_id` in the name.
 if ENV['BUILD_NUMBER'] then
     $build_id_name = "-build-#{$build_id}"
-    $rsync_exclude = ".git"
-else
-    $rsync_exclude = "GIT_VERSION"
 end
 
 if ENV['K8S'] then
-    $k8stag = ENV['K8STAG'] || "-k8s"
+    $vm_base_name = "k8s"
+else
+    $vm_base_name = "runtime"
 end
 
-# We need this workaround since kube-proxy is not aware of multiple network
-# interfaces. If we send a packet to a service IP that packet is sent
-# to the default route, because the service IP is unknown by the linux routing
-# table, with the source IP of the interface in the default routing table, even
-# though the service IP should be routed to a different interface.
-# This particular workaround is only needed for cilium, running on a pod on host
-# network namespace, to reach out kube-api-server.
-$kube_proxy_workaround = <<SCRIPT
-sudo iptables -t nat -A POSTROUTING -o enp0s8 ! -s 192.168.34.12 -j MASQUERADE
-SCRIPT
-
 Vagrant.configure(2) do |config|
     config.vm.provision "bootstrap", type: "shell", inline: $bootstrap
     config.vm.provision "build", type: "shell", run: "always", privileged: false, inline: $build
     config.vm.provision "install", type: "shell", run: "always", privileged: false, inline: $install
 
-    config.vm.provider :libvirt do |libvirt|
-        config.vm.box = "cilium/ubuntu-16.10"
-        config.vm.box_version = "2.0"
-        libvirt.memory = ENV['VM_MEMORY'].to_i
-        libvirt.cpus = ENV['VM_CPUS'].to_i
-        config.vm.synced_folder ".", "/home/vagrant/go/src/github.com/cilium/cilium", disabled: false
-    end
-
     config.vm.provider "virtualbox" do |vb|
         # Do not inherit DNS server from host, use proxy
         vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
         vb.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
-
-        config.vm.box = "cilium/ubuntu-16.10"
-	config.vm.box_version = "2.7"
+        config.vm.box = "eloycoto/ubuntu_test"
+        config.vm.box_version = "22"
         vb.memory = ENV['VM_MEMORY'].to_i
         vb.cpus = ENV['VM_CPUS'].to_i
 
-        if ENV["NFS"] then
-            config.vm.synced_folder '.', '/home/vagrant/go/src/github.com/cilium/cilium', type: "nfs"
-            # Don't forget to enable this ports on your host before starting the VM
-            # in order to have nfs working
-            # iptables -I INPUT -p udp -s 192.168.34.0/24 --dport 111 -j ACCEPT
-            # iptables -I INPUT -p udp -s 192.168.34.0/24 --dport 2049 -j ACCEPT
-            # iptables -I INPUT -p udp -s 192.168.34.0/24 --dport 20048 -j ACCEPT
-        else
```

Comment:
I'm using NFS as part of my workflow, so I would like to see the "NFS" option to remain.

Reply:
NFS is being removed because we now install Guest Additions into the VM, which means that syncing of the filesystem will happen automatically.

Reply:
Great!